app/routes/vote/challenge-extension-ideas.js

```diff
@@ -0,0 +1,8 @@
+import Route from '@ember/routing/route';
+
+export default class VoteChallengeExtensionIdeasRoute extends Route {
```

Comment:
@Arpan-206 why do we need these files? The route name is `course-extension-ideas`, and we already have a route file for it. This seems like it's not used at all?